k8s/upgrades/0.8.0-0.8.1/cstor-volume-patch.tpl.json

```diff
@@ -0,0 +1,10 @@
+{
+   "metadata": {
+      "annotations": {
+         "openebs.io/storage-class-ref": "name: @sc_name\nresourceVersion: @sc_resource_version\n"
```

Comment:
is this annotation good?
One key 'storage-class-ref' having again multiple keys in it, which will cause problem for any later processing..
Same applies to other places as well
any thoughts @AmitKumarDas ?